openstack/cinder/templates/etc/_ratelimit.yaml.tpl

```diff
@@ -0,0 +1,99 @@
+{{ if .Values.api_rate_limit.enabled -}}
+{{ if .Values.api_rate_limit.project_whitelist -}}
+# List of whitelisted scopes keys (domainName/projectName).
+whitelist: {{ .Values.api_rate_limit.project_whitelist }}
+{{- end }}
+{{- end }}
+
+# Override default ratelimit response.
+ratelimit_response:
+  status: 498 Rate Limited
+  status_code: 498
```

Comment:
is this the returned HTTP status code?   Shouldn't it be 429 ?
429 Too Many Requests

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
Please see: https://github.com/sapcc/helm-charts/pull/2149#issuecomment-748096203

Reply:
You can set it as you like. Swift uses 498, but 429 is the standard. I think it's a good idea to enforce http standard

Reply:
set to 429 now